runtime/CMakeLists.txt

```diff
@@ -268,6 +270,8 @@ set(OMR_PORT_LIB "j9prt" CACHE INTERNAL "")
 set(OMR_THREAD_LIB "j9thr" CACHE INTERNAL "")
 set(OMR_TRACE_LIB "j9trc" CACHE INTERNAL "")
 
+set(OMR_DDR_SET "j9ddr" CACHE INTERNAL "")
```

Comment:
Is this logically part of the `# setup OMR library overrides` section?  Or a new section?

Reply:
its somewhat in between it is an override of a default omr property, but it isn't a library in the strict sense of the term